src/js/components/modals/__tests__/InstallPackageModal-test.js

```diff
@@ -63,7 +70,7 @@ describe('InstallPackageModal', function () {
         this.instance.getModalContents(),
         this.container
       ));
-      var result = node.querySelectorAll('p')[1];
+      var result = node.querySelectorAll('p')[4];
```

Comment:
this is because we're actually rendering the schema form now, which will itself take up some nodes. therefore we need to look deeper